core/metadata.jvm/test/org/jetbrains/kotlin/metadata/jvm/JarContentTest.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.metadata.jvm
+
+import jdk.internal.org.objectweb.asm.ClassReader
```

Comment:
Please use `org.jetbrains.org.objectweb.asm` instead

Reply:
I wasn't aware of differences, thanks!